sdk/keyvault/keyvault-common/package.json

```diff
@@ -5,18 +5,19 @@
   "sdk-type": "client",
   "author": "Microsoft Corporation",
   "license": "MIT",
-  "main": "dist/index.js",
-  "module": "dist-esm/src/index.js",
-  "types": "types/keyvault-common.d.ts",
+  "main": "./dist/commonjs/index.js",
+  "browser": "./dist/browser/index.js",
```

Comment:
Not sure whether to leave browser support or remove it. On the one hand, lack of CORS support from keyvault means most browsers will refuse to work with KV but on the other hand:

- We don't know if anyone is using this in their bundler
- We don't know if anyone is using this with cors protection disabled (very bad but not unheard of in karma tests at least)

So maybe we remove browser support for common/keys/certs/secrets as a major bump or as part of typespec migration? 

Alternatively, we can say that faking browser support is a bug that is now being fixed - not sure what the right path is...

@timovv what do you think? Any preference?

Reply:
Yeah, good point. Thinking about it that way it's probably better to keep it in, even if we aren't testing it -- it's unsupported, but removing it means there's a risk of breaking people, even if the risk is small, and it's not a risk we really need to take. I don't think it's worth doing a major version bump for this alone, but if we do end up doing new major at some point it would probably be worth rolling it in to that.